ide/maven.rst

```diff
@@ -0,0 +1,15 @@
+Maven
+=====
+
+Eirslett
+--------
+
+.. hint::
+    You can disable the maven-frontend-plugin by disabling the eirslett plugin, either directly or by passing the
+    ``skipEirslett`` argument. Use ``-DskipEirslett`` in the command line to do this.
+
+We use `frontend-maven-plugin`_ to install npm for the new client and run gulp to compile Less / Sass and the likes.
```

Comment:
it's more the other way around: to install the new client with npm :)

Reply:
suggestion :
We use `frontend-maven-plugin`_ to run various npm tasks such as installing js polyfill packages or packages of the new client and run gulp to compile Less / Sass and the likes.

Reply:
I like it, gonna use your version